pyvisa_py/serial.py

```diff
@@ -182,25 +182,23 @@ def write(self, data: bytes) -> Tuple[int, StatusCode]:
         end_out, _ = self.get_attribute(ResourceAttribute.asrl_end_out)
         send_end, _ = self.get_attribute(ResourceAttribute.send_end_enabled)
 
-        try:
-            if end_out in (SerialTermination.none, SerialTermination.termination_break):
```

Comment:
@MatthieuDartiailh Why did you remove the check for `end_out` matching `SerialTermination.none` here? This means I fall into the `else` later and so my writes fail now with `Unknown value for VI_ATTR_ASRL_END_OUT: SerialTermination.none`

Reply:
Good catch. I was trying to avoid an empty branch but messed it up. I will fix it later today. Regarding the termchar the story is that for serial instrument sending the termchar can be part of the communication protocol itself while in othrr cases it is only required for the message to be meaningful. We could try avoiding duplicating it but I am unsure of the consequences.